android/tangram/src/main/cpp/jniExports.cpp

```diff
@@ -367,7 +370,7 @@ extern "C" {
         assert(mapPtr > 0);
         auto map = reinterpret_cast<Tangram::Map*>(mapPtr);
         auto sourceName = stringFromJString(jniEnv, name);
-        auto source = std::shared_ptr<Tangram::TileSource>(new Tangram::ClientGeoJsonSource(sourceName, ""));
+        auto source = std::shared_ptr<Tangram::TileSource>(new Tangram::ClientGeoJsonSource(map->getPlatform(), sourceName, ""));
```

Comment:
Ahh.. but I am not sure if [this](https://github.com/tangrams/tangram-es/blob/168903c9feb5cdd22e40c09e79123320707e23d5/android/tangram/src/main/cpp/jniExports.cpp#L111) can be abstracted out within a `Map` function.